dvc-requirements.txt

```diff
@@ -0,0 +1,4 @@
+dvc==2.38.1
+pandas
```

Comment:
Could we install the additional packages (panda, scikit-learn and SQLAlchemy) using the pipeline generated requirements file? As additional virtualenv.run() command after 'git init' and 'dvc init'?